### PR TITLE
Cairo u96 implementation

### PR DESCRIPTION
**Pull request by PhilippeR26**
## Motivation and Resolution

Cairo has introduced a new type : `u96`
In an abi, it's identified as a non trivial description : `core::internal::bounded_int::BoundedInt::<0, 79228162514264337593543950335>` _No comments 🧐_
It's represented as a number in the calldata.

## Usage related changes

u96 is now handled and can be manipulated as a BigNuberish.

## Development related changes

u96 is handled as a litteral.

## Checklist:

- [x] Performed a self-review of the code
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Documented the changes in code (API docs will be generated automatically)
- [x] Updated the tests
- [x] All tests are passing

### CODE REVIEW

File: __tests__/cairov24onward.test.ts [hunk 1]
```diff
@@ -375,4 +375,36 @@ describe('Cairo v2.4 onwards', () => {
       await expect(nonZeroContract.call('send_nonZero_u256', [myU512])).rejects.toThrow();
     });
   });
+
+  describe('Cairo v282 u96', () => {
```
Comment (ivpavici) [inline __tests__/cairov24onward.test.ts:379]: just a nitpick for consistency, other describe-s have a different version format (with dots), like 'Cairo **v2.6.0** ...'

Comment (PhilippeR26) [reply __tests__/cairov24onward.test.ts]: Modified.

Comment (PhilippeR26) [reply __tests__/cairov24onward.test.ts]: Modified.

**Review by ivpavici — APPROVED** [review]

**Review by tabaktoni — APPROVED** [review]
WP LGTM!

**Pull request merged**